### PR TITLE
Create strings.xml

### PR DESCRIPTION
**Pull request by Tntdruid**
Added Danish translation.

### CODE REVIEW

File: opentasks/src/main/res/values-da/strings.xml [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    <!-- Task detail view -->
+    <string name="activity_task_details_edit">Edit</string>
+    <string name="activity_task_details_edit_task">Rediger opgave</string>
+    <string name="activity_task_details_delete">Slet</string>
+    <string name="activity_task_details_delete_task">Slet opgave</string>
+    <string name="activity_task_details_complete">Komplet</string>
+    <string name="activity_task_details_complete_task">Komplet opgave</string>
+    <string name="activity_task_details_pin">Pin</string>
+    <string name="activity_task_details_pin_task">Pin opgave</string>
+    <string name="opentasks_details_no_task_selected">Ingen opgave valgt/string>
```
Comment (dmfs) [inline opentasks/src/main/res/values-da/strings.xml]: Some of the strings, like this one are not properly terminated, resulting in an error (see https://travis-ci.org/dmfs/opentasks/builds/596938209#L2690).

**Comment by dmfs** [comment]
Thank you for the translation. Looks like there are a few missing `<` in the data. Can you add those?

File: opentasks/src/main/res/values-da/strings.xml (hunk 1 above)
Comment (Tntdruid) [reply opentasks/src/main/res/values-da/strings.xml]: Fixed the missing <

**Comment by dmfs** [comment]
Thank you!

**Comment by Tntdruid** [comment]
@dmfs np 👍

**Pull request merged**